docs/readmes/lte/migration_guide.md

```diff
@@ -0,0 +1,38 @@
+---
+id: migration_guide
+title: AGW migration guide
+hide_title: true
+---
+
+# Access Gateway migration guide
+
+> NOTE: Only Ubuntu-based Access Gateways are supported starting from Magma release 1.6, Debian support is deprecated.
+
+For a smooth transition from a Debian to Ubuntu access gateway, it is advised to backup local configs including `RootCA`, `control_proxy.yml`, `Hardware ID` and `Challenge Key` files. It is also recommended to backup any static IPs or routes provisioned on the existing access gateway.
+
+### Before migration
+
+#### Copy the contents from the following file locations to a local backup storage:
+
+- `Control Proxy`
+> */var/opt/magma/configs/control_proxy.yml*
```

Comment:
For this and the ones below

Also, put them in the same order as the previous section

```suggestion
- Control proxy config: `/var/opt/magma/configs/control_proxy.yml`
```